[CI] Update expected result files

desertfire opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #169860

Summary: benchmarks/dynamo/ci_expected_accuracy/update_expected.py stopped working because it needs to deal with both cuda and rocm runs now. Fixed it and updated the result files for inductor-periodic jobs.

cc @jeffdaily @sunway513 @jithunnair-amd @pruthvistony @ROCmSupport @jataylo @hongxiayang @naromero77amd @pragupta @jerrymannil @xinyazhang @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @kadeng @chauhang @amjames @Lucaskabela

desertfire committed:
[CI] Update expected result files

Summary: benchmarks/dynamo/ci_expected_accuracy/update_expected.py stopped working because it needs to deal with both cuda and rocm runs now. Fixed it and updated the result files for inductor-periodic jobs.

[ghstack-poisoned]

zou3519 reviewed (commented):
why does it look like some models are being deleted?

desertfire committed:
Update on "[CI] Update expected result files"


Summary: benchmarks/dynamo/ci_expected_accuracy/update_expected.py stopped working because it needs to deal with both cuda and rocm runs now. Fixed it and updated the result files for inductor-periodic jobs.

cc jeffdaily sunway513 jithunnair-amd pruthvistony ROCmSupport jataylo hongxiayang naromero77amd pragupta jerrymannil xinyazhang voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Update expected result files"


Summary: benchmarks/dynamo/ci_expected_accuracy/update_expected.py stopped working because it needs to deal with both cuda and rocm runs now. Fixed it and updated the result files for inductor-periodic jobs.

cc jeffdaily sunway513 jithunnair-amd pruthvistony ROCmSupport jataylo hongxiayang naromero77amd pragupta jerrymannil xinyazhang voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Update expected result files"


Summary: benchmarks/dynamo/ci_expected_accuracy/update_expected.py stopped working because it needs to deal with both cuda and rocm runs now. Fixed it and updated the result files for inductor-periodic jobs.

cc jeffdaily sunway513 jithunnair-amd pruthvistony ROCmSupport jataylo hongxiayang naromero77amd pragupta jerrymannil xinyazhang voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Update expected result files"


Summary: benchmarks/dynamo/ci_expected_accuracy/update_expected.py stopped working because it needs to deal with both cuda and rocm runs now. Fixed it and updated the result files for inductor-periodic jobs.

cc jeffdaily sunway513 jithunnair-amd pruthvistony ROCmSupport jataylo hongxiayang naromero77amd pragupta jerrymannil xinyazhang voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Update expected result files"


Summary: benchmarks/dynamo/ci_expected_accuracy/update_expected.py stopped working because it needs to deal with both cuda and rocm runs now. Fixed it and updated the result files for inductor-periodic jobs.

cc jeffdaily sunway513 jithunnair-amd pruthvistony ROCmSupport jataylo hongxiayang naromero77amd pragupta jerrymannil xinyazhang voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

desertfire commented:
@pytorchbot merge
